docs/cmd/kn_broker_create.md

```diff
@@ -23,6 +23,7 @@ kn broker create NAME
 ```
       --backoff-delay string     The delay before retrying.
       --backoff-policy string    The retry backoff policy (linear, exponential).
+      --broker-config string     Broker config object like ConfigMap or RabbitMQ
```

Comment:
Somehow this statement is missing something "Broker config object like ConfigMap or RabbitMQ" can we be more descriptive. Also is "or" the right conjunctive here?

Reply:
I updated the flag description in my latest commit. Please let me know if it can be improved further